content/tracing/advanced_usage/priority_sampling.md

```diff
@@ -40,9 +40,31 @@ public class MyClass {
 ```
 {{% /tab %}}
 {{% tab "Python" %}}
+To enable sampling priority in the tracer:
```

Comment:
Reword in: "Priority sampling is disabled by default. To enable it, configure the `priority_sampling` flag using the tracer configuration method:"